content/docs/command-reference/metrics/add.md

```diff
@@ -18,42 +18,36 @@ defines the given `path` as an <abbr>output</abbr>, marking `path` as a metric
 file to track.
 
 Note that outputs can also be marked as metrics via the `-m` or `-M` options of
-`dvc run`.
+`dvc run`. We recommend using `-M` option to keep metrics in Git history.
 
-While any text file can be tracked as a metric file, we recommend using TSV,
-CSV, or JSON formats. DVC provides a way to parse those formats to get to a
-specific value, if the file contains multiple metrics. See the
-[options](#options) below and `dvc metrics show` for more info.
+While any text file can be tracked as a metric file, we recommend using JSON
+formats. DVC provides a way to parse this formats to get to a specific value, if
+the file contains multiple metrics. See the [options](#options) below and
+`dvc metrics diff` for more info.
 
 > Note that [external output](/doc/user-guide/managing-external-data) cannot be
 > marked as project metrics.
 
 ## Options
 
 - `-t <type>`, `--type <type>` - specify a type for the metric file. Accepted
-  values are: `raw` (default), `json`, `tsv`, `htsv`, `csv`, `hcsv`. It will be
-  saved into the corresponding DVC-file, and used by `dvc metrics show` to
-  determine how to handle displaying metrics.
+  values are: `raw` (default), `json`. It will be saved into the corresponding
+  DVC-file, and used by `dvc metrics show` to determine how to handle displaying
+  metrics.
 
   `raw` means that no additional parsing is applied, and `--xpath` is ignored.
```

Comment:
This paragraph is pretty lonely now. Maybe say what `json` is or just append it to the previous p?
> Same in metrics modify and show cmd refs